src/Checkbox.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+import { Icon, Flex, Box, Label, theme } from '..'
+
+class Checkbox extends React.Component {
+  handleClick(e) {
+    const { name, value, onClick } = this.props
+    const checked = e.target.checked
+    onClick(name, value, checked)
+  }
+
+  render() {
+    const {
+      name,
+      id,
+      disabled,
+      size,
+      fontSize,
+      children,
+      onClick,
+      checked,
+      defaultChecked,
+      theme: themeProp,
+      ...restProps
+    } = this.props
+    const currentTheme = themeProp || theme
+    return (
+      <StyledLabel
+        htmlFor={name}
+        data-test="design-system-checkbox"
+        disabled={disabled}
+      >
+        <input
+          type="checkbox"
+          name={name}
+          id={this.props.id || name}
+          disabled={disabled}
+          defaultChecked={checked || defaultChecked}
```

Comment:
We should probably keep the props passed in to the `<input>` the same, it seems like this would affect how `checked` and `defaultChecked` work, right?